contracts/scripts/staking/transferStakes.js

```diff
@@ -0,0 +1,53 @@
+// Script for verifying against etherscan.
+//
+// Usage:
+//  - Run:
+//      node signStakingPayout.js <PrivateKey> <fromAddress> <toAddress> <r> <s> <v>
+//
+//
+//
+const { ethers } = require("hardhat");
+const { utils, BigNumber, Wallet } = require("ethers");
+const { getTxOpts } = require("../../utils/tx");
+const fs = require("fs");
+
+const TEST_AGENT_PK =
+  "0x345c8d05224b66bab10e9f952dc1d332e59e062be5990f87206a67e4545e132d";
+
+async function doTransfer(pk, fromAddress, toAddress, r, s, v) {
+  const wallet = new Wallet(pk, ethers.provider);
+  const proxyAddress = (await ethers.getContract("OGNStakingProxy")).address;
+  const contract = await ethers.getContractAt(
+    "SingleAssetStaking",
+    proxyAddress
+  );
+
+  const ops = await getTxOpts();
+  ops.gasLimit = 6000000;
+  ops.gasPrice = 9000000000;
```

Comment:
With the introduction of EIP 1559, I think ehter.js by default sends type 2 transaction and setting the gas price is ignored.
I would just remove all this and only set gasLimit in the call to transferStakes below.
like 
```
.transferStakes(fromAddress, toAddress, r, s, v, { gasLimit: 9000000000);
```